notebooks/video_clips/yolo4.py

```diff
@@ -0,0 +1,201 @@
+import sys
+import cv2
+import argparse
+import random
+import time
+import os
+import os.path as osp
+
+import urllib3
+from pathlib import Path
+import hashlib
+from tqdm import tqdm
+
+# source: https://github.com/kingardor/YOLOv4-OpenCV-CUDA-DNN
+class YOLOv4:
+
+    def __init__(self,args=None):
+        """ Method called when object of this class is created. """
+
+        self.args = None
+        self.net = None
+        self.names = None
+
+        self.args = self.parse_arguments() if args is None else args
+        self.initialize_network()
+        self.run_inference()
+
+    def parse_arguments(self):
+        """ Method to parse arguments using argparser. """
+
+        parser = argparse.ArgumentParser(description='Object Detection using YOLOv4 and OpenCV4')

```

Comment:
Is there a reason to add this style of arguments ? Is this intended to be used as a CLI in future?